example/py-shiny/example_pyshiny_reactive_scatter.py

```diff
@@ -0,0 +1,57 @@
+import matplotlib.pyplot as plt
+import seaborn as sns
+from shiny import App, reactive, render, ui
+
+from maidr.custom_renderers.shiny_renderer import render_maidr
+
+# Load the dataset
+iris = sns.load_dataset("iris")
+
+# Define the UI components for the Shiny application
+app_ui = ui.page_fluid(
+    ui.row(
+        ui.column(
+            3,
+            ui.input_select(
+                "x_var",
+                "Select X variable:",
+                choices=iris.select_dtypes(include=["float64"]).columns.tolist(),
+                selected="sepal_length",
+            ),
+            ui.input_select(
+                "y_var",
+                "Select Y variable:",
+                choices=iris.select_dtypes(include=["float64"]).columns.tolist(),
+                selected="sepal_width",
+            ),
+        ),
+        ui.column(9, ui.output_ui("reactivebarplot")),
+    )
+)
+
+
+# Define the server
+def server(input, output, session):
+    @reactive.Calc
+    def create_reactivebarplot():
+        fig, ax = plt.subplots(figsize=(10, 6))
+        s_plot = sns.scatterplot(
+            data=iris, x=input.x_var(), y=input.y_var(), hue="species", ax=ax
+        )
+        ax.set_title(f"Iris {input.y_var()} vs {input.x_var()}")
+        ax.set_xlabel(input.x_var().replace("_", " ").title())
+        ax.set_ylabel(input.y_var().replace("_", " ").title())
+        return s_plot
+
+    @output
+    @render.ui
```

Comment:
This plot has been injected into the shiny through `ui.output_ui`. Why `@output` is required?

Reply:
`@output` is for rendering reactive output on the UI but since `@render.ui` is sufficient in our case to render the generated HTML on the shiny app, `@output` is redundant. Will have this removed.